nym-api/src/support/config/template.rs

```diff
@@ -1,6 +1,8 @@
 // Copyright 2021 - Nym Technologies SA <contact@nymtech.net>
 // SPDX-License-Identifier: GPL-3.0-only
 
+// TODO dz any changes to this ?
```

Comment:
yes. as I had on my branch, you need to put the bind address into the config because rocket pulls it from `Rocket.toml` which obviously won't exist